go/go-sql/README.md

```diff
@@ -7,28 +7,29 @@ SQLcommenter is a plugin/middleware/wrapper to augment application related infor
 ### Install from source
 
 * Clone the source
-* In terminal go inside the client folder location where we need to import google-sqlcommenter package and enter the below commands
+* In terminal go inside the client folder location where we need to import sqlcommenter go-sql module and enter the below commands
 
 ```shell
-go mod edit -replace google.com/sqlcommenter=path/to/google/sqlcommenter/go
+go mod edit -replace google.com/sqlcommenter=path/to/google/sqlcommenter/go-sql
 
 go mod tiny
+
+go get google.com/sqlcommenter/gosql
 ```
 ### Install from github [To be added]
 
-## Usages
+## Usage
 
-### go-sql-driver
-Please use the sqlcommenter's default database driver to execute statements. \
+Please use the sqlcommenter's default go-sql database driver to execute statements. \
 Due to inherent nature of Go, the safer way to pass information from framework to database driver is via `context`. So, it is recommended to use the context based methods of `DB` interface like `QueryContext`, `ExecContext` and `PrepareContext`. 
 
 ```go
 db, err := sqlcommenter.Open("<driver>", "<connectionString>", sqlcommenter.CommenterOptions{<tag>:<bool>})
```

Comment:
Maybe change it to sqlcommentersql.Open ?
As they are not a single package now.